docs/bib/butler-bandpasses.bib

```diff
@@ -0,0 +1,10 @@
+@misc{10.71929/rubin/2587407,
+    author = "{NSF-DOE Vera C. Rubin Observatory}",
+    doi = "10.71929/RUBIN/2587407",
+    url = "https://www.osti.gov//servlets/purl/2587407",
+    keywords = "79 ASTRONOMY AND ASTROPHYSICS",
+    title = "{Legacy Survey of Space and Time Data Preview 1: bandpasses dataset type [Data set]}",
```

Comment:
```suggestion
    title = "{Legacy Survey of Space and Time Data Preview 1: standard_bandpass dataset type [Data set]}",
```
(thanks for reminding me I needed to update the lsst-texmf repo)

Reply:
Ah! Hadn't realized that word wasn't just descriptive but needed to match the Butler dataset type but that makes sense -- committing suggestion thanks